### ISSUE TITLE
false positive SA1629.

### ISSUE THREAD

**Issue by msmolka** [DotNetAnalyzers/StyleCopAnalyzers#2679]
I have remarks with code:

```xml

        /// <remarks>
        /// Sample request:
        /// <code>
        ///     DELETE api/v1/regions/f788b478-b0df-e511-80ee-000c2933216e
        /// </code>
        /// </remarks>

```

And stylecop is complaining about.

**Comment by sharwell** [comment]
@vweijsters @pdelvo I'm thinking we should leave this rule disabled by default, but allow projects to manually enable it. The overall number of false positives when I tried to enable it in StyleCop Analyzers left me uncomfortable.

**Comment by vweijsters** [comment]
I'm OK with disabling it by default.

For the StyleCop Analyzers projects, I haven't seen any false positives, unless you consider the `<seealso>` tags as being exempt from this rule.

I would prefer to enable this rule for StyleCop Analyzers at some point in the future.

**Comment by RalfKoban** [comment]
I have the same problem here with `<seealso>` tags.

Following snippet produces a (imho) false-positive:
` /// <seealso href="https://docs.microsoft.com/en-us/dotnet/framework/wpf/index">Windows Presentation Foundation</seealso>`

**Comment by sharwell** [comment]
@vweijsters The `<seealso>` tags would be a good start for sure
